### ISSUE TITLE
[Feat request] Option to shut down computer + keep audio and segments calculation files

### ISSUE THREAD

**Issue by Anacondo** [porcino/Av1ador#13]
Hola! Could you please add an option to keep the audio and segments calculations after encoding is finished? Sometimes I need to adjust the CRF values and run a 2nd encode to hit a target file size, and it would save a lot of time if I didn't have to calculate the chunks + encode the audio again. Also, could you add an option to shutdown the computer after the queue is finished? Thanks a lot!

**Comment by Anacondo** [comment]
Just the .txt file with the segments and the audio file, no need for the .ivf files as the encode will start over anyway. Also, would it be possible to override the confirmation to use the existing files from a previous encode, so that you don't have to be in front of the PC to resume an old/previous encode? Thanks a lot!

**Comment by porcino** [comment]
I added a cleanup button to choose what files to keep.

> would it be possible to override the confirmation to use the existing files from a previous encode, so that you don't have to be in front of the PC to resume an old/previous encode?

Now this dialog will not appear if you enable the button "encode entire list".
The only missing thing is the option to shutdown the computer, I don't know where to put it.

**Comment by Anacondo** [comment]
> I added a cleanup button to choose what files to keep.
> Now this dialog will not appear if you enable the button "encode entire list", default is always resume

These work wonderfully, thank you so much!

> The only missing thing is the option to shutdown the computer, I don't know where to put it.

May I suggest something like this?

![image](https://github.com/porcino/Av1ador/assets/5350602/2f0fa234-b75e-468d-8200-7448f87c8ad4)

**Comment by Anacondo** [comment]
I think I found a couple bugs:

1. The settings for the cleanup button seem to reset every time I change the selected video in the list, it doesn't remember my settings unless I close the program first, at which point it writes the selected option to the settings.xml file. If I choose my preferred option, close the program and open it again it works as expected.

2. Dialog to resume no longer appears, as expected, when encode entire list is selected. However, it seems that no matter what options are selected in the cleanup, all files present in the temp folder will be erased and the encode restarted from scratch. I've experimented with different cleanup options but it seems to make no difference.
